CHANGELOG_TEMPLATE.md

```diff
@@ -0,0 +1,21 @@
+OpenHospital VERSION
+---------------------
+
+Changelog VERSION:
+
+CHANGELOGCORE
+
+CHANGELOGGUI
```

Comment:
I think in this template we could just have the "Changelog VERSION" (manually written summary) and the two CHANGELOG can be saved in doc/ as full `git log` (as it is now, for offline consultation), while in this template we could replace the placeholders with the links only (like you suggested [once](https://github.com/informatici/openhospital-core/compare/v1.9.1...master)), for all involved components (now only "core" and "gui", in the future also "api" and "ui").

The aim is to reduce the verbosity of the resulting CHANGELOG.md

Otherwise it is OK for me